meta-resin-common/recipes-connectivity/openssh/balena-files/sshd.socket

```diff
@@ -1,10 +1,10 @@
 [Unit]
-Conflicts=dropbear.service
+Conflicts=sshd.service
 
 [Socket]
+ExecStartPre=/bin/mkdir -p /var/run/sshd
 ListenStream=22222
 Accept=yes
 
 [Install]
 WantedBy=sockets.target
-Also=dropbearkey.service
```

Comment:
I'm guessing `sshdgenkeys.service` replaces `dropbearkey.service`.
But I don't see `Also=sshdgenkeys.service`.. I'm guessing the OS still works and the service still runs to generate keys..

 otherwise ssh would have been broken.

Reply:
We install that separately as I didn't want to change the upstream service that much. See `SYSTEMD_SERVICE_${PN}-sshd += "sshdgenkeys.service"`.

Reply:
:+1: